packages/connector-naver/src/constant.ts

```diff
@@ -0,0 +1,29 @@
+import { ConnectorMetadata, ConnectorPlatform } from '@logto/connector-core';
+
+export const authorizationEndpoint = 'https://nid.naver.com/oauth2.0/authorize';
+export const accessTokenEndpoint = 'https://nid.naver.com/oauth2.0/token';
+export const userInfoEndpoint = 'https://openapi.naver.com/v1/nid/me';
+
+export const defaultMetadata: ConnectorMetadata = {
+  id: 'naver-universal',
+  target: 'naver',
+  platform: ConnectorPlatform.Universal,
+  name: {
+    en: 'Naver',
+    'zh-CN': 'Naver',
+    'tr-TR': 'Naver',
+    'ko-KR': '네이버',
+  },
+  logo: './logo.svg',
+  logoDark: null,
+  description: {
+    en: 'Naver is the most famous internet service provider in South Korea',
+    'zh-CN': 'Naver is the most famous internet service provider in South Korea',
+    'tr-TR': 'Naver is the most famous internet service provider in South Korea',
+    'ko-KR': '네이버는 한국에서 가장 유명한 인터넷 서비스 제공자 입니다.',
```

Comment:
nit: btw, after a team discussion, we prefer not to use words like 'the most', 'the biggest' since the situation could change. The rewording of other connectors' descriptions will be applied soon.
Maybe we can use the word 'leading'.